src/c_api/c_api.cc

```diff
@@ -97,9 +97,31 @@ inline int MXAPIGetFunctionRegInfo(const FunRegType *e,
 }
 
 // NOTE: return value is added in API_END
+/*!
+ * \brief Convert NDArray to C type arrays for passing to custom library
+ */
+void NDArrayToCTypeArray(const std::vector<NDArray>& inputs,
```

Comment:
here we're converting a vector of NDArrays to vectors of C types, so shouldnt the name be "NDArrayToCTypes"?

Reply:
agree